security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -836,12 +841,13 @@ func (sc *SecretCache) sendRetriableRequest(ctx context.Context, csrPEM []byte,
 		if isCSR {
 			requestErrorString = fmt.Sprintf("%s CSR", conIDresourceNamePrefix)
 			certChainPEM, err = sc.fetcher.CaClient.CSRSign(
-				ctx, csrPEM, exchangedToken, int64(sc.configOptions.SecretTTL.Seconds()))
+				ctx, reqID, csrPEM, exchangedToken, int64(sc.configOptions.SecretTTL.Seconds()))
 		} else {
-			requestErrorString = fmt.Sprintf("%s token exchange", conIDresourceNamePrefix)
+			requestErrorString = fmt.Sprintf("%s TokExch", conIDresourceNamePrefix)
 			p := sc.configOptions.Plugins[0]
 			exchangedToken, _, httpRespCode, err = p.ExchangeToken(ctx, sc.configOptions.TrustDomain, exchangedToken)
 		}
+		cacheLog.Debugf("%s", requestErrorString)
```

Comment:
remove? we are already logging it

Reply:
But we are only logging it when an error happens.  I think this is OK since it's Debugf :)